Examples/Tutorials/Tutorial-DpdkL2Fwd/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+add_executable(Tutorial-DpdkL2Fwd main.cpp WorkerThread.cpp)
+
+target_link_libraries(Tutorial-DpdkL2Fwd PUBLIC PcapPlusPlus::Pcap++)
+
+if(PCAPPP_INSTALL)
+  install(
+    TARGETS Tutorial-DpdkL2Fwd
+    EXPORT PcapPlusPlusTargets
+    RUNTIME DESTINATION ${PCAPPP_INSTALL_BINDIR})
+endif()
```

Comment:
I'm not sure we should install the tutorials, do you think users will benefit from it? 🤔

Reply:
no strong opinion on this up to you.

Reply:
In that case I'd suggest we don't install the tutorials